examples/deepddi_examples.py

```diff
@@ -0,0 +1,28 @@
+"""Example with DeepDDI."""
+
+from chemicalx import pipeline
+from chemicalx.data import DrugbankDDI
+from chemicalx.models import DeepDDI
+
+
+def main():
+    """Train and evaluate the DeepSynergy model."""
+    dataset = DrugbankDDI()
+    model = DeepDDI(drug_channels=dataset.drug_channels, hidden_layers_num=2)
```

Comment:
why does the number of hidden layers in the example differ from the chosen default?

Reply:
It is intended to make the training process faster.

Reply:
I think we want the examples to show configurations that get good results. Does this model take a long time to run on this dataset?

Reply:
It runs slowly on just CPUs. You're right and I will change it to the default settings as same as the original paper.